### PR TITLE
feat: add property `forceScroll` to `CCard`

### PR DESCRIPTION
**Pull request by ItsPierre**

### CODE REVIEW

File: src/components/CCard.vue [hunk 1]
```diff
@@ -150,6 +152,13 @@ const props = withDefaults(
      */
     loading?: boolean;
 
+    /**
+     * If true, the card will always be in scrollable mode.
+     *
+     * This adds the dividers and always shows the scroll bar.
+     */
+    scrollable?: boolean;
```
Comment (johannes-huther) [inline src/components/CCard.vue]: Should we name this `forceScroll` or something?

File: src/components/CCard.vue [hunk 2]
```diff
@@ -40,15 +40,17 @@
...
 
     <v-card-text
       ref="cardTextComponent"
       class="ma-0 px-0"
-      :class="scrollingContentVerticalPadding && scrollable ? 'py-4' : 'py-0'"
-      style="overflow-y: auto"
+      :class="
+        scrollingContentVerticalPadding && scrollingEnabled ? 'py-4' : 'py-0'
+      "
+      :style="scrollingEnabled ? 'overflow-y: scroll' : 'overflow-y: auto'"
```
Comment (johannes-huther) [inline src/components/CCard.vue]: I believe this should be `scrollable ? 'overflow-y: scroll'...` technically, shouldn't it?

File: src/components/CCard.vue [hunk 3]
```diff
@@ -48,7 +48,9 @@
       ref="cardTextComponent"
       class="ma-0 px-0"
       :class="
-        scrollingContentVerticalPadding && scrollingEnabled ? 'py-4' : 'py-0'
+        scrollingContentVerticalPadding && scrollingEnabled
+          ? 'pt-4 pb-0'
+          : 'py-0'
```
Comment (johannes-huther) [inline src/components/CCard.vue:53]: I don't quite understand, but isn't the bottom padding 0 in both cases? How does that fit the commit message
> fix: remove bottom padding of card content when scrollable

**Review by johannes-huther — CHANGES_REQUESTED** [review]

File: src/components/CCard.vue (hunk 3 above)
Comment (ItsPierre) [reply src/components/CCard.vue:53]: no, before the change, the padding was `py-4`, if I understand you correct

File: src/components/CCard.vue (hunk 2 above)
Comment (ItsPierre) [reply src/components/CCard.vue]: sounds reasonable

**Comment by ItsPierre** [comment]
https://github.com/Crystal-Creations-GbR/crystal-components/compare/48a57c6b430614a031a3ff9189cea7699c759958..141f1b2375bb2f7e470ac042706d06d84aed4f12

**Comment by ItsPierre** [comment]
Chromatic Screenshots seem to have a problem with displaying any scrollbar. For now, we ignore this, but might have a look at this at some point.

File: src/components/CCard.vue (hunk 3 above)
Comment (johannes-huther) [reply src/components/CCard.vue:53]: Yeah, now that you say it, it does actually make sense...

File: src/components/CCard.vue (hunk 1 above)
Comment (johannes-huther) [reply src/components/CCard.vue]: Please fix the story also

**Review by johannes-huther — APPROVED** [review]

**Pull request merged**